blog/models.py

```diff
@@ -1,3 +1,19 @@
 from django.db import models
 
 # Create your models here.
+
+class Blog(models.Model):
+    title = models.CharField(max_length=100)
```

Comment:
could add `unique=True` (would require another migration) to let the DB enforce unique blog post titles

Reply:
I like that idea! Where would that line of code go?

Reply:
`title = models.CharField(max_length=100, unique=True)`

Reply:
... then run `python manage.py makemigrations` to make the corresponding migration file (and `git add` it to commit), then run `python manage.py migrate` to sync that change to the DB